src/test/isolation2/expected/frozen_insert_crash.out

```diff
@@ -0,0 +1,226 @@
+-- Test server crash in case of frozen insert. Make sure that after crash
+-- recovery, the frozen insert and index are consistent:
+--
+-- 1. If crash happened before the row is frozen, the row will be invisible;
+-- 2. If crash happened after the row is frozen, the row will be visible.
+--
+-- And the above behavior should remain consistent using seqscan or indexscan.
+--
+-- We test gp_fastsequence here since it does frozen insert and has an index.
+
+-- Case 1. crash after the regular MVCC insert has made to disk, but not
+-- the WAL record responsible for updating it to frozen.
+-- After crash recovery, the insert will follow regular MVCC and not be seen.
+1: create table tab_fi(a int) with (appendoptimized=true) distributed replicated;
+CREATE
+
+-- switch WAL on seg0 to reduce flakiness
+1: select gp_segment_id, pg_switch_xlog() is not null from gp_dist_random('gp_id') where gp_segment_id = 0;
+ gp_segment_id | ?column? 
+---------------+----------
+ 0             | t        
+(1 row)
+
+2: insert into tab_fi select generate_series(1, 1000);
```

Comment:
The original test file has no this instructions. Why do we need them?

Reply:
To switch to frozen sequences.